pyfar/dsp/filter/fractional_octaves.py

```diff
@@ -329,6 +371,90 @@ def _coefficients_fractional_octave_bands(
     return sos
 
 
+def check_fractional_octave_band_filter_tolerance(
+            fractional_octave_band_filter, num_fractions, frequency_range,
+            tolerance_class):
+    """
+    Check magnitude tolerance of fractional octave band filters.
+
+    The tolerance is defined in [#]_ and returned by
+    :py:func:`~pyfar.constants.fractional_octave_filter_tolerance`.
+    See :py:func:`~pyfar.dsp.filter.fractional_octave_bands` for an example.
+
+    Parameters
+    ----------
+    fractional_octave_band_filter : FilterSOS
+        The fractional-octave-band filters.
+    num_fractions : int
+        The number of bands an octave is divided into. Must be ``1`` (octave
+        bands) or ``3`` (one-third octave bands).
+    frequency_range : array, tuple
+        The lower and upper frequency limits.
+    tolerance_class : int
+        The tolerance class as defined in the standard. Must be ``1`` or ``2``.
+
+    Returns
+    -------
+    tolerance_met : bool
+        ``True`` if the fractional octave band filters meet the tolerance,
+        ``False`` otherwise.
+
+    References
+    ----------
+    .. [#] IEC 61260-1 (2014). Octave-band and fractional-octave-band filters.
+           Part 1: Specifications.
+    """
+
+    if type(fractional_octave_band_filter) is not pf.FilterSOS:
+        raise TypeError('fractional_octave_bands must be a FilterSOS object')
+
+    if num_fractions not in [1, 3]:
+        raise ValueError('num_fractions must be 1 or 3')
+
+    if tolerance_class not in [1, 2]:
+        raise ValueError('tolerance_class must be 1 or 2')
+
+    exact_center_frequencies = fractional_octave_frequencies(
+        num_fractions, frequency_range)[1]
+
+    ir = fractional_octave_band_filter.impulse_response()
```

Comment:
would it make sence to directly insert the impulse_response of the fitler instead of the Filterobject?
otherwise we could bypass **kwargs to ``fractional_octave_band_filter.impulse_response(**kwargs)``